cloud/transport/cluster.go

```diff
@@ -98,10 +99,15 @@ func (c *Cluster) Open() error {
 	return nil
 }
 
+func (c *Cluster) Shutdown() {
+	log.Printf("server shutdown")
```

Comment:
Did you intend to leave this here? If so, might be helpful to include more context in the log, like which server is shutting down (might not be obvious when intermingled with all other logs in the console) server address, etc

Reply:
Yes I intended to leave it. We don't call it anywhere yet and I'm 99% we don't have proper shutdown routine for a whole raft cluster. It was called in tests so that's why I added a new one. Once we go around handling shutdown this will probably get refined.